papers/donath/donath-pylira.rst

```diff
@@ -0,0 +1,423 @@
+:author: Axel Donath
+:email: axel.donath@cfa.harvard.edu
+:institution: Center for Astrophysics | Harvard & Smithsonian
+:orcid: 0000-0003-4568-7005
+
+:author: Aneta Siemiginowska
+:email: asiemiginowska@cfa.harvard.edu
+:institution: Center for Astrophysics | Harvard & Smithsonian
+:orcid: 0000-0002-0905-7375
+
+:author: Vinay Kashyap
+:email: vkashyap@cfa.harvard.edu
+:institution: Center for Astrophysics | Harvard & Smithsonian
+:orcid: 0000-0002-3869-7996
+
+:author: Douglas Burke
+:email: dburke@cfa.harvard.edu
+:institution: Center for Astrophysics | Harvard & Smithsonian
+
+:author: Karthik Reddy Solipuram
+:institution: University of Maryland Baltimore County
+
+:author: David van Dyk
+:institution: Imperial College London
+
+:bibliography: mybib
+
+:video: -
+
+----------------------------------------------------------------
+Pylira: deconvolution of images in the presence of Poisson noise
+----------------------------------------------------------------
+
+.. class:: abstract
+
+    All physical and astronomical imaging observations are degraded by the limited angular
+    resolution of the camera and telescope systems. The recovery of the true image is limited by
+    both how well the instrument characteristics are known and by the magnitude of measurement noise.
+    In the case of a high signal to noise ratio data, the image can be sharpened or “deconvolved” robustly
+    by using established standard methods such the Richardson-Lucy method. However the situation changes for rare
+    sparse data and the low signal to noise regime, where deconvolution leads inevitably to an amplification
+    of noise and poorly reconstructed images. However the results in this regime can be improved
+    by making use of prior assumptions. One proposed method is the LIRA algorithm, which
+    requires smoothness of the reconstructed image at multiple scales. In this contribution we
+    introduce a new python package called *Pylira*, which exposes the original C implementation
+    of the LIRA algorithm to Python users. We briefly describe the package structure, development
+    setup and show a Chandra as well as Fermi-LAT analysis example.
+
+
+
+.. class:: keywords
+
+   deconvolution, point spread function, poisson, low counts, x-ray, gamma-ray
+
+Introduction
+------------
+Any physical and astronomical imaging process is affected by the limited
+angular resolution of the instrument or telescope. In addition the quality
+of the resulting image is also degraded by background or instrumental
+measurement noise and non-uniform exposure.
+For short wavelengths and associated low intensities
+of the signal, the imaging process consists in recording individual photons arriving from
```

Comment:
consist of recording individual photons _(often called "events")_ arriving from a source of interest.

Reply:
Done